### PR TITLE
Delete old git.NewCommand() and use it as git.NewCommandContext()

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: modules/git/git.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func LoadGitVersion() error {
 		return nil
 	}
 
-	stdout, err := NewCommand("version").Run()
+	stdout, err := NewCommandContext(context.Background(), "version").Run()
```
Comment (6543) [inline modules/git/git.go]: since it's exec by init i used context.Background ...

**Comment by KN4CK3R** [comment]
I propose to use only `NewCommand` because otherwise every method will end in `...Context` sometimes. As the context parameter should be the new normal, we could drop the `Context` suffix.

**Comment by zeripath** [comment]
Also need to get rid of the commands like CloneWithContext etc.

**Comment by 6543** [comment]
@zeripath can not find `CloneWithContext` and I think it's for a next pull :)

**Comment by zeripath** [comment]
Looks like I might have fixed that in the previous pr.

**Comment by lunny** [comment]
> I propose to use only `NewCommand` because otherwise every method will end in `...Context` sometimes. As the context parameter should be the new normal, we could drop the `Context` suffix.

I think so. `NewCommand` is a better name than `NewCommandContext`.

**Review by singuliere — APPROVED** [review]

**Comment by 6543** [comment]
@lunny already changed :D

**Review by lunny — APPROVED** [review]

**Comment by KN4CK3R** [comment]
Title should be changed.

**Comment by 6543** [comment]
@KN4CK3R suggestions for a better one are welcome

**Comment by KN4CK3R** [comment]
I would just have switched both methods but the current one is ok too. 👍 

> replace git.NewCommandContext() with git.NewCommand()

**Pull request merged**